docker-compose.yml

```diff
@@ -152,16 +152,18 @@ services:
       - "traefik.http.routers.api.rule=Host(`api.metacpan.localhost`)"
       - "traefik.http.services.api.loadbalancer.server.port=5000"
 
-  api_test:
+  apitest:
     depends_on:
       - elasticsearch_test
+      - pgdb
```

Comment:
Are there any tests which use Pg? Right now I think Minion will default to SQLite.

Reply:
Unknown, but as the container is going to be up anyway, don't see a downside to having it available.

Reply:
Ok. If it incurs some kind of a penalty, we can revisit this.